libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ErrorCohort, ILabeledStatistic } from "@responsible-ai/core-ui";
+import { PointOptionsObject } from "highcharts";
+import { IDropdownOption } from "office-ui-fabric-react";
+
+export function generateCohortsStatsTable(
+  cohorts: ErrorCohort[],
+  selectableMetrics: IDropdownOption[],
+  labeledStatistics: ILabeledStatistic[][],
+  selectedMetrics: string[]
+) {
+  const items: PointOptionsObject[] = cohorts.map(
+    (errorCohort, cohortIndex) => {
+      return {
+        colorValue: 0,
+        value: errorCohort.cohortStats.totalCohort,
+
+        x: 0,
+        // metric index for Count column
+        y: cohortIndex
+      };
+    }
+  );
+  selectableMetrics
+    .filter((element: IDropdownOption) =>
+      selectedMetrics.includes(element.key.toString())
+    )
+    .forEach((metricOption: IDropdownOption, metricIndex: number) => {
+      // first determine min and max values
+      let metricMin = Number.MAX_SAFE_INTEGER;
+      let metricMax = Number.MIN_SAFE_INTEGER;
+      cohorts.forEach((_errorCohort, cohortIndex) => {
+        const stat = labeledStatistics[cohortIndex].find(

```

Comment:
nit: maybe call it metricStat instead of stat, so instead of stat.stat you will have metricStat.stat to reduce confusion when looking at the code